src/main/java/seedu/duke/command/CommandAddProperty.java

```diff
@@ -1,3 +1,5 @@
+//@@author OVReader
```

Comment:
From what I remember, the first line should always be package. 
You might want to check back in your code if there is any warnings or errors.